src/en/anymovie/src/eu/kanade/tachiyomi/animeextension/en/anymovie/AnyMovie.kt

```diff
@@ -22,11 +22,11 @@ import org.jsoup.nodes.Element
 import uy.kohesive.injekt.Injekt
 import uy.kohesive.injekt.api.get
 
-class AllMovies : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
+class AnyMovie : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
 
-    override val name = "AllMoviesForYou"
+    override val name = "AnyMovie"
 
-    override val baseUrl = "https://allmoviesforyou.net"
+    override val baseUrl = "https://anymovie.cc/"
```

Comment:
```suggestion
    override val baseUrl = "https://anymovie.cc"
```
Remove trailing slash